tests/test_full_text_search.py

```diff
@@ -362,17 +380,36 @@ def test_search_by_name_only(self):
         results = full_text_search.search('haiti',
                                           {'name': 'Rin Shibuya',
                                            'location': 'Shinjuku Rin'}, 5)
-        assert not results            
+        assert not results
 
         # Input the name and location in the wrong box
         results = full_text_search.search('haiti',
                                           {'name': 'Shinjuku',
                                            'location': 'Rin Shibuya'}, 5)
-        assert not results         
+        assert not results
+
+        # Search by Special Chinese Family Name
+        # while records are written in English
+        results = full_text_search.search('haiti', {'name': u'单鱼'}, 5)
+        assert set([r.record_id for r in results]) == \
```

Comment:
It seems like in many tests, we use ' \ ' to break the line, I am wondering is there any better way to break line without using ' \ ' in this case?
Is something like
```
assert (.... ==
         ....)
```
better?